playbooks/roles/deploy-osh/tasks/component-install.yml

```diff
@@ -1,21 +1,39 @@
 ---
-- name: "Prepare override file {{ _cmpnt_template }}"
-  config_template:
-    src: "{{ _cmpnt_template }}.j2"
-    dest: "/tmp/socok8s-{{ _cmpnt_template }}"
-    config_overrides: "{{ _cmpnt_overrides }}"
-    config_type: yaml
-  register: _usertemplatedata
+# Example structure
+#_cmpnt_runcommand: ./tools/deployment/multinode/020-ingress.sh
+#_cmpnt_overrides:
+#  - suse: ingress-namespace.yaml
+#    user: "{{ suse_osh_deploy_ingress_namespace_yaml_overrides }}"
+#    envvars_getting_those_overrides:
+#      - "OSH_EXTRA_HELM_ARGS_INGRESS_OPENSTACK"
```

Comment:
Can't we extract these envvars from the override files rather than having to define them separately?

Reply:
What does this bring? I think it would mask some thing and therefore be harder to understand?

Reply:
Maybe I'm confused by their purpose, but aren't these defined within the supplied override.suse/user files? If so, aren't we repeating them here?

Reply:
the overrides is just template generation. We need to say where and how to apply the overrides. This is done through env vars. So basically the script x can be passed env vars y and z getting the overrides for components y and z.

Reply:
Should be clearer now.